app/main/reducers/location-reducers.js

```diff
@@ -10,21 +10,16 @@
  specific language governing permissions and limitations under the License.
  */
 
-import * as profileCommandTypes from '../../shared/constants/profile-command-types';
+import * as profileActionTypes from '../constants/profile-action-types';
 
 import Immutable from 'immutable';
 
-export default function locationReducer(state, command) {
-  if (typeof state === 'undefined') {
-    return Immutable.Map();
-  }
+export default function locationReducer(state = new Immutable.Map(), action) {
+  const payload = action.payload;
```

Comment:
This'll need to be rebased on top of the new, better fake results at: https://github.com/mozilla/tofino/commit/8bd95b5a8da91eff7cf66f2807f73d9d6dd84189#diff-2918c2484fc2347d14dfdfdcb2d8bf1bL24